pkg/kubelet/config/apiserver.go

```diff
@@ -14,7 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Reads the pod configuration from the Kubernetes apiserver.
```

Comment:
Curiosity question... how did this comment violate the golinter? Or did you just decide it wasn't necessary?

Either is fine, just curious :)

Reply:
Hmm, when I first ran it, golint gave me an error because every package should only have one package level comment (basically a comment above the `package` statement). However, if I try to reproduce it (by putting comments back into the top of the files) now, I don't get  any errors. If you want I can put them back in. However, I think we should rather come up with a true package-level comment instead of comments describing what the file does.

Reply:
I don't think we need this comment here, the methods in this file are commented.

the package itself should be documented though.